app/controllers/edit_phone_controller.rb

```diff
@@ -63,7 +63,7 @@ def code_send
   def verify
     state = MultiFactorAuth.verify_code(current_user, params[:phone_code])
     if state == :ok
-      old_phone = current_user.phone
+      old_phone = current_user.phone || "nothing"
```

Comment:
Is this for the benefit for the `record_security_event` below?
Because there'll be a nil case and it's better this reads `from nothing`

Reply:
Yes